vectordatasource/transform.py

```diff
@@ -1291,12 +1295,57 @@ def _f(shape, cutting_shape):
     return _f
 
 
+# we can load external geojson layers from file, all we need is a directory and geojson file named after the layer
+# we expect it to already be in mercator projection and we only fill out the minimal structure of the feature tuple
+# there's no ID, most of the datum stuff is missing, we are mostly worried about the shape and properties
+# we return None if we couldnt load the layer for any reason
+def _load_external_layer(params, name):
```

Comment:
this is moving over to `tilequeue` where the rest of the layers are loaded and ill add a new section to queries.yaml instead of having it be part of the transform